source/Calamari/Deployment/PackageRetention/Model/Journal.cs

```diff
@@ -28,24 +29,28 @@ public void RegisterPackageUse(PackageIdentity package, ServerTaskId serverTaskI
         {
             try
             {
-                if (repository.TryGetJournalEntry(package, out var entry))
+                using (var repository = repositoryFactory.CreateJournalRepository())
```

Comment:
You can use the alternative `using` syntax here, and save some braces, i.e:
`using var repository = repositoryFactory.CreateJournalRepository();`
The repository will be disposed of at the end of the current block.
See info here: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement

Reply:
I'm getting this in Rider:
![image](https://user-images.githubusercontent.com/11970672/141875313-5cc9a37f-4cd0-4470-8413-163f83d7f680.png)

Reply:
That's weird, Calamari is c# 8.0 project for me.  Feel free to go back to the old syntax, it's no big deal.
![image](https://user-images.githubusercontent.com/46470837/141888532-dc89f21b-396a-4460-bd65-d9f6f3e8ff92.png)

Reply:
🤔 Some of the projects like `Calamari.Common` are C# 8.0, no idea why the `Calamari` one is stuck on C# 7.3 for me.